dashboard/test/controllers/api/v1/pd/course_facilitators_controller_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class Api::V1::Pd::CourseFacilitatorsControllerTest < ::ActionController::TestCase
+  test 'admins can see course facilitators' do
+    sign_in create(:admin)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :success
+  end
+
+  test 'workshop organizers can see course facilitators' do
+    sign_in create(:workshop_organizer)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :success
+  end
+
+  test 'facilitators cannot see course facilitators' do
+    sign_in create(:facilitator)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :forbidden
+  end
+
+  test 'teachers cannot see course facilitators' do
+    sign_in create(:teacher)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :forbidden
```

Comment:
I tried this and it does not work as expected. Calling sign_in multiple times in a single test case leaves the original user logged in. Subsequent calls to the API still run under the original user context. This is true even if I call sign_out user in between. There's probably a way to do this, but it's not obvious to me at the moment - plus with Laurel's point about clear failure messaging I'm going to punt the test optimization.